src/Media_Command.php

```
@@ -2452,4 +2452,803 @@ private function update_post_content_for_attachment( array $url_replacements ) {
+
+		if ( $limit > 0 && count( $candidates ) > $limit ) {
+			$candidates = array_slice( $candidates, 0, $limit );
+		}
+
+		if ( empty( $candidates ) && 'table' === $format ) {
+			WP_CLI::success( 'No orphan media found.' );
+			return;
+		}
+
+		$fields_raw = Utils\get_flag_value( $assoc_args, 'fields' );
+		$fields     = null === $fields_raw ? $allowed_fields : explode( ',', $fields_raw );
```

PHPStan (2.2.7)

[error] Parameter `#2` $string of function explode expects string, mixed given.


(argument.type)